test/Serialization/Address.purs

```diff
@@ -0,0 +1,44 @@
+module Test.Serialization.Address(suite) where
+
+import Prelude
+
+import Data.Maybe (Maybe, maybe, fromJust)
+import Effect.Class (liftEffect)
+import Effect.Exception (error, throwException)
+import Partial.Unsafe (unsafePartial)
+import TestM (TestPlanM)
+import Serialization.Address (Bech32String(..), addressBech32, addressNetworkId, addressPubKeyHash, addressStakeKeyHash, fromBech32, newBaseAddress)
+
+
+errBool :: String -> Boolean -> TestPlanM Unit
```

Comment:
`assertTrue` would be a better name I think. Can we plug `Test.Unit.Assert.*` assertions here BTW?